otterdog/eclipse-basyx.jsonnet

```diff
@@ -257,6 +257,23 @@ orgs.newOrg('eclipse-basyx') {
         default_workflow_permissions: "write",
       },
     },
+    orgs.newRepo('basyx-pdf-to-aas') {
+      allow_merge_commit: true,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      web_commit_signoff_required: false,
+      description: "Python library to exctract technical data via LLMs from PDF and export them in AAS format.",
```

Comment:
There's a typo in extract

Reply:
Thats right, thank you for your feedback Frank. Its corrected now.